src/pytrajplot/plotting/plot_map.py

```diff
@@ -1,12 +1,19 @@
 """Generate Map Plot Figure."""
 
 # Standard library
+from itertools import groupby
 from pathlib import Path
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
```

Comment:
The base collections are now generic since python 3.9 and thus using the typing proxy objects is deprecated.
More info here: https://peps.python.org/pep-0585/